tools/windows/install-help/install-cmd/install-cmd.vcxproj.filters

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ClCompile Include="cmdline.cpp" />
+    <ClCompile Include="install-cmd.cpp" />
+    <ClCompile Include="stdafx.cpp" />
+    <ClCompile Include="..\cal\caninstall.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\customactiondata.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\ddreg.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\FinalizeInstall.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\stopservices.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\PropertyReplacer.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\strings.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
+    <ClCompile Include="..\cal\TargetMachine.cpp">
+      <Filter>cal_imported</Filter>
+    </ClCompile>
```

Comment:
So you deleted distribution and creation of install-cmd.exe but project is remained. Is this exe useful?